doc/Language/faq.pod6

```diff
@@ -299,6 +299,69 @@ dependencies within a single compilation unit (e.g., file) are possible
 through stubbing. Therefore another possible solution is to move
 classes into the same compilation unit.
 
+=head2 System: How can I work with file and folder
+
```

Comment:
Not a well formulated question. Also, there's no issue requesting it, there's a [whole tutorial page for input/output](https://docs.raku.org/language/io), as well as the pages of the respective utilities. Let's please keep FAQs for FAQs, and [in this issue](https://github.com/Raku/doc/issues/3109) most of them were already covered.